framework/framework_cancel_crud_test.go

```diff
@@ -102,59 +97,9 @@ func Test_ProcessDelete_CRUD(t *testing.T) {
 			},
 			ErrorMatcher: nil,
 		},
-		// Test 4 ensures ProcessDelete executes all resources in the
-		// expected order when reconciliationcanceledcontext is not set.
-		{
-			Ctx: context.Background(),
-			Resources: []Resource{
-				newTestCRUDResource("r0"),
-				newTestCRUDResource("r1"),
-				newTestCRUDResource("r2").CancelReconciliationAt("GetDesiredState"),
-				newTestCRUDResource("r3"),
-				newTestCRUDResource("r4"),
-			},
-			ExpectedOrder: []string{
-				"r0.GetCurrentState",
-				"r0.GetDesiredState",
-				"r0.NewDeletePatch",
-				"r0.ApplyCreateChange",
-				"r0.ApplyDeleteChange",
-				"r0.ApplyUpdateChange",
-
-				"r1.GetCurrentState",
-				"r1.GetDesiredState",
-				"r1.NewDeletePatch",
-				"r1.ApplyCreateChange",
-				"r1.ApplyDeleteChange",
-				"r1.ApplyUpdateChange",
-
-				"r2.GetCurrentState",
-				"r2.GetDesiredState",
-				"r2.NewDeletePatch",
-				"r2.ApplyCreateChange",
-				"r2.ApplyDeleteChange",
-				"r2.ApplyUpdateChange",
-
-				"r3.GetCurrentState",
-				"r3.GetDesiredState",
-				"r3.NewDeletePatch",
-				"r3.ApplyCreateChange",
-				"r3.ApplyDeleteChange",
-				"r3.ApplyUpdateChange",
-
-				"r4.GetCurrentState",
-				"r4.GetDesiredState",
-				"r4.NewDeletePatch",
-				"r4.ApplyCreateChange",
-				"r4.ApplyDeleteChange",
-				"r4.ApplyUpdateChange",
-			},
-			ErrorMatcher: nil,
-		},
```

Comment:
There is no such case now, which I believe was confusing and inconsistent with resource cancellation.